projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -994,13 +994,25 @@ export abstract class IgxGridBaseDirective implements GridType,
     public gridCopy = new EventEmitter<IGridClipboardEvent>();
 
     /**
-     * @hidden @internal
+     * Emitted when the rows are expanded or collapsed.
+     *
+     * @example
+     * ```html
+     * <igx-grid [data]="employeeData" (selectedRowsChange)="selectedRowsChange($event)" [autoGenerate]="true"></igx-grid>
```

Comment:
The description says:
`(selectedRowsChange)="selectedRowsChange($event)"` but this is for `expansionStatesChange` event

Reply:
fixed